app/assets/javascripts/settings/ldap-conn-test.js

```diff
@@ -0,0 +1,76 @@
+$(function () {
+  function noneEmpty (arr) {
+    for (var i = 0; i < arr.length; i++) {
+      if (arr[i] === '') return false
+    }
+    return true
+  };
+
+  function renderValidity (msg, isValid) {
+    $('#ldap_conn_message').removeClass('ldap-conn-message-default ldap-conn-message-valid ldap-conn-message-invalid')
+    if (isValid) {
+      $('#ldap_conn_save').removeProp('disabled')
+      $('#ldap_conn_message').addClass('ldap-conn-message-valid')
+    } else {
+      $('#ldap_conn_save').prop('disabled', 'disabled')
+      $('#ldap_conn_message').addClass('ldap-conn-message-invalid')
+    }
+    $('#ldap_conn_message').html(msg)
+  }
+
+  $(document).ready(function () {
```

Comment:
This is already inside of a `ready` callback, `$(function() { ...` it's a shortcut for this.

Reply:
Fixed now.